### DISCUSSION TITLE
從不懂 Python 變成了 AI 工程師

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#814] [General]
文章中說明了作者如何從其他領域跨進機器學習領域並慢慢成為專家，中間受到挫折又是怎麼解決的，最終又收穫了甚麼。

我從他初學自學到後來受專業教育知曉，若對一項事情有興趣學習就馬上去做，受挫折就找其他方法解決，他受學術訓練後依舊須從面試獲取教訓與經驗，這都告訴我們要不斷學習，不是所有事情都會照著我們所認為的方向去走，失敗要化為學習的動力，AI不僅是要會應用實作，懂得我們使用的模型背後的數學知識還有分析也是相當重要的。

https://fc.bnext.com.tw/articles/view/454

**Comment by Doreen331** [comment]
嗨同學，你的網址點不進去呦

**Reply by ching020721** [reply]
謝謝提醒，已經修正了，有興趣可以點進去看看~

**Comment by ru0372572** [comment]
現在還是點不進去!

**Reply by ching020721** [reply]
感謝提醒，網址有做修正了，若對文章細節有興趣可以再看看!

**Comment by Pietra1226** [comment]
程式語言吸引人的地方就是其實自學也可以有很快的進步，也讓高科技公司有各種不同管道可以吸引人才。這是一個比較不用在乎學歷，而在乎實作的產業

**Comment by Kotori0501** [comment]
我上大學一開始也是python零基礎，剛開始學真的有點困難，但自從接觸到圖像辨識的python我就覺得很有趣，所以就開始對程式有興趣了。

**Comment by yuxinlin93** [comment]
這個故事也太勵志了吧，他強大的決心跟行動力很值得學習

**Comment by WEI-CHEN-CHUN** [comment]
每一件事情都是從不回開始，上大學才學程式絕對不會太晚，最重要的是學了Python想要做些甚麼

**Comment by dabbie109303501** [comment]
個人覺得在學程式的過程中，嘗試從零開始寫一個小專案對學習很有幫助

**Comment by Chiway1229** [comment]
我覺得學習最重要的就是永不放棄和勇於嘗試，只要有個想實現的目標就不是難題
希望我的python之路也能找到一個想完成的目標

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
萬事起頭難 但只要堅持下去 相信未來不管面臨什麼問題都能迎刃而解

**Comment by dinosaur1009** [comment]
學習程式真的需要蠻多debug的耐心 還有沒有想法的挫折 但如果撐過前面適應期 感覺就好很多了

**Comment by Zoe06060606** [comment]
許多事難的都不是事情 而是心態 只要下定決心 堅持下去 離成功就不遠了

**Comment by BlahFish1006** [comment]
好勵志的故事，謝謝分享！讓現在對學習寫程式越來越灰心喪志的自己又獲得力量了~
大家一起加油><

**Comment by manipulateed** [comment]
堅持自己的路不忘初衷，以及熱愛這個領域正是能努力不懈的源頭吧～也是給我們一個很好很有啟發性的歷程，使我們對於接下來的挑戰無所畏懼

**Comment by CPCpyt** [comment]
感謝分享！學習程式語言真的是一大趨勢，不論是任何領域在和程式結合後都有令人意想不到的發展。

**Comment by StevenWanglolz** [comment]
看完我覺得好佩服喔，從零開始學了這麼多東西，也從每一次的面試吸取經驗，還繼續深造，我已五體投地...

**Comment by ghost** [comment]
一個很勵志的故事，人在面對不確定性很高的事物，向來會選擇拖延或逃避，然文章中的這位，從發現感興趣的事物時，卻能放下較擅長的事物，選擇追尋自己所愛，這份勇氣，值得學習

**Comment by Evan0979** [comment]
非常勵志!希望自已能向他學習,以後成為某個領域的專家!
